### DISCUSSION TITLE
GitLens icon is strange in my vscode, any fix?

### DISCUSSION THREAD

**Discussion by j0k4rX** [gitkraken/vscode-gitlens#1515] [Q&A]
![Screenshot from 2021-05-20 10-54-17](https://user-images.githubusercontent.com/71499522/118921720-7705dc00-b95a-11eb-8c17-00be00afb7cc.png)

**Comment by eamodio** [comment]
Yeah, unfortunately there was a change in VS Code that stops the correct icon from showing up, I'm hoping there will be a VS Code "fix" for that soon.
